pkg/machine/e2e/list_test.go

```diff
@@ -70,6 +70,34 @@ var _ = Describe("podman machine list", func() {
 		Expect(util.StringInSlice(name1, listNames)).To(BeTrue())
 		Expect(util.StringInSlice(name2, listNames)).To(BeTrue())
 	})
+
+	It("list machine: check if running while starting", func() {
+		i := new(initMachine)
+		session, err := mb.setCmd(i.withImagePath(mb.imagePath)).run()
+		Expect(err).To(BeNil())
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
I know I dont do it all the time, but i think @edsantiago prefers we use Expect(session).To(Exit(0))  ... @edsantiago can you confirm that?

Reply:
Good catch @baude, thank you. Yes, please use the `.To(Exit(0))` form. It yields a slightly more helpful failure message, which can save someone time in the future.